packages/EasyStandard/docs/services/sniffs.md

```diff
@@ -0,0 +1,410 @@
+     *
+     * @param int $withSomething
+     * 
+     * @return void
+     */
+    public function doSomethingC(int $withSomething): void
+    {
+
+    }
+    
+    /**
+     * Do something.
+     *
+     * @param bool $withSomething
+     * 
+     * @return void
+     */
+    public function doSomethingD(bool $withSomething): void
+    {
+
+    }
+}
+```
+
+#### Control Structures
+
+##### [\EonX\EasyStandard\Sniffs\ControlStructures\ArrangeActAssertSniff][7]
+Checks that test method must conform to Arrange, Act and Assert (AAA) Pattern. Allow amount of empty lines is between [1, 2].
+```php
+// incorrect
+final class TestClass
+{
+    public function testSomethingA()
+    {
+        $expectedResult = 4;
+        $array = [
+            'key' => 'value',
+        ];
+        $actualResult = 2 + 2;
+        self::assertSame($expectedResult, $actualResult);
+        self::assertSame(['key' => 'value'], $array);
+    }
+    
+    public function testSomethingB()
+    {
+        $expectedResult = 4;
+        $actualResult = 2 + 2;
+        self::assertSame($expectedResult, $actualResult);
+    }
+}
+```
+```php
+// correct
+final class TestClass
+{
+    public function testSomethingA()
+    {
+        $expectedResult = 4;
+        $array = [
+            'key' => 'value',
+        ];
+
+        $actualResult = 2 + 2;
+
+        self::assertSame($expectedResult, $actualResult);
+        self::assertSame(['key' => 'value'], $array);
+    }
+    
+    public function testSomethingB()
+    {
+        $actualResult = 2 + 2;
+        
+        self::assertSame(4, $actualResult);
+    }
+
+    public function testSomethingC()
+    {
+        self::assertSame(4, 2 + 2);
+    }    
+
+    // Allow empty line in closure
+    public function testSomethingD()
+    {
+        $value1 = 2;
+        $value2 = 2;
+        $expectedClosure = static function () use ($value1, $value2): int {
+            $result = $value1 + $value2;
+
+            return $result + 0;
+        };
+
+        $actualResult = 2 + 2;
+
+        self::assertSame($expectedClosure(), $actualResult);
+    }
+
+    public function noTestMethod()
+    {
+        $expectedResult = 4;
+        $actualResult = 2 + 2;
+        self::assertSame($expectedResult, $actualResult);
+    }
+}
+```
+**Parameters**
+
+- `testMethodPrefix` - If the method name starts with this prefix, checks will be applied to it. Default value: `test`.
+- `testNamespace` - If the class namespace begins with this prefix, the class will be parsed. Default value: `App\Tests`.
+```php
+// correct
+namespace App\NoTestNamespace;
+
+final class TestClass
+{
+    public function testSomething()
+    {
+        $expectedResult = 4;
+
+        $actualResult = 2 + 2;
+
+        self::assertSame($expectedResult, $actualResult);
+
+        echo $actualResult;
+    }
+}
+```
+
+##### [\EonX\EasyStandard\Sniffs\ControlStructures\NoNotOperatorSniff][8]
+Strict comparison operator should be used instead NOT operator.
```

Comment:
`A strict comparison operator must be used instead of a NOT operator.`

Reply:
Done.